libpng in this tree does not ship the lsx sources,
so disable them to avoid undefined symbols

--- a/media/libpng/pngpriv.h
+++ b/media/libpng/pngpriv.h
@@ -221,7 +221,7 @@
 
 #ifndef PNG_LOONGARCH_LSX_OPT
 #  if defined(__loongarch_sx)
-#     define PNG_LOONGARCH_LSX_OPT 1
+#     define PNG_LOONGARCH_LSX_OPT 0
 #  else
 #     define PNG_LOONGARCH_LSX_OPT 0
 #  endif
